### PR TITLE
Exception Dialogs

### PR DESCRIPTION
**Pull request by jakobmoellerdev**

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -554,6 +554,11 @@
 
     <string name="submission_result_button_dialog_close">OK</string>
 
+    <!-- ####################################
+               Generic Error Messages
+        ###################################### -->
+    <string name="errors_storage_headline">Something went wrong</string>
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values/strings.xml:560]: please use german as default language

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -554,6 +554,11 @@
...
+    <string name="errors_storage_button_positive">Continue</string>
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values/strings.xml:561]: same here

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DialogHelper.kt [hunk 3]
```diff
@@ -7,14 +7,34 @@ object DialogHelper {
 
     data class DialogInstance(
         val activity: Activity,
-        val title: Int,
-        val message: Int,
-        val positiveButton: Int,
-        val negativeButton: Int? = null,
-        val cancelable: Boolean = true,
+        val title: String,
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DialogHelper.kt:10]: are the other dialogs still working with this change?

**Review by harambasicluka — CHANGES_REQUESTED** [review]

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 2 above)
Comment (jakobmoellersap) [reply Corona-Warn-App/src/main/res/values/strings.xml]: done

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 1 above)
Comment (jakobmoellersap) [reply Corona-Warn-App/src/main/res/values/strings.xml]: done

**Review by harambasicluka — DISMISSED** [review]

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**